### PR TITLE
Fix/utc 1722 repair toggle submenu

### PR DESCRIPTION
**Pull request by UTCbridget**
When Section Editors are logged in, the submenu was not behaving properly. While we got the submenu to show upon page load with a previous PR, the toggle button still did not work in prod or test, and in local or dev when the web profiler module is turned off:

toggle side navigation submenus not working

For some reason conditional statements and toggles are not applying the classes to the parent when clicking .more indicator. The click fires, but nothing happens.

toggle side navigation submenus not working no classes

So I did a work around with changing the click function to a delegate function and added classes to the .more based on the state of the menu.

toggle side navigation submenus

Also, I changed the cursor to pointer when hovering over the .more indicator.

### CODE REVIEW

**Review by UTCGilligan — APPROVED** [review]

**Comment by UTCGilligan** [comment]
OK @UTCbridget make the UTCCloud PR and we'll get this moving into Dev.

**Comment by UTCbridget** [comment]
Don't I have to wait until this PR is merged into develop? https://github.com/UTCWeb/utccloud/wiki/Syncing-Particle-and-UTCCloud-PRs

**Pull request merged**
